data/items/items.xml

```diff
@@ -17590,6 +17590,7 @@
 	</item>
 	<item id="9975" article="a" name="crystal table">
 		<attribute key="weight" value="2500" />
+		<attribute key="wrapTo" value="9974" />
```

Comment:
@wgrriffel regarding your previous comment ```because items that can be wrapped is a type not an attribute, its more easier to deal with it in lua and makes more sense, since we got more similar itemtypes such as isTeleport and isTile, why not isWrappable.``` I can't check if item is wrappable. It requires either items.xml attribute or otb flag (can also be done in lua lib folder if I get @marksamman's permission).